Mathlib/NumberTheory/ModularForms/CongruenceSubgroups.lean

```diff
@@ -148,7 +148,7 @@ def Gamma1' (N : ℕ) : Subgroup (Gamma0 N) :=
   (Gamma0Map N).ker
 #align Gamma1' Gamma1'
 
-@[simp] -- Porting note: linter failed to synth `CommMonoid { x // x ∈ Gamma0 N }`
+@[simp]
```

Comment:
This error *looks* different (but presumably is fine to remove/perhaps this was more obvious with the original change).